packages/terminal/package.json

```diff
@@ -7,7 +7,7 @@
     "@theia/filesystem": "^0.3.17",
     "@theia/process": "^0.3.17",
     "@theia/workspace": "^0.3.17",
-    "xterm": "~3.8.0"
+    "xterm": "~3.9.1"
```

Comment:
maybe we shouldn't use caret there (as we only have CQ for 3.9.1) even if yarn.lock is using 3.9.1 (but it might be another discussion)

Reply:
thanks!

Reply:
@svenefftinge you use ranges for external dependencies that are not coming from Eclipse projects ?

Reply:
Thanks @marcdumais-work! Looking forward to read on the outcome of that meeting.